homeassistant/helpers/entity_registry.py

```diff
@@ -161,6 +161,7 @@ class RegistryEntry:
     previous_unique_id: str | None = attr.ib(default=None)
     aliases: set[str] = attr.ib(factory=set)
     area_id: str | None = attr.ib(default=None)
+    categories: dict[str, str] = attr.ib(factory=dict)
```

Comment:
Same question as above: this can just be a string ID to the category that belongs to the domain ?